packages/@tinacms/schema-tools/src/types.ts

```diff
@@ -534,6 +534,7 @@ export interface Config<
   admin?: {
     auth?: {
       onLogin?: (args: { token: TokenObject }) => Promise<void>
+      onLogout: () => Promise<void>
```

Comment:
should this be optional?

Reply:
yes it should

Reply:
Fixed (I fixed it yesterday but forgot to push it up)